exdb/tests/integration_tests.py

```diff
@@ -546,7 +547,6 @@ def test_week_ahead(self):
                                               start_datetime=(now() + timedelta(days=2)),
                                               end_datetime=(now() + timedelta(days=3)),
                                               type=self.create_type(),
-                                              subtype=self.create_subtype(),
```

Comment:
You should probably still have a subtype set on this experience to make it closer to representing the real thing.